src/plopp/backends/matplotlib/canvas.py

```diff
@@ -90,9 +73,10 @@ def __init__(
         figsize: tuple[float, float] | None = None,
         title: str | None = None,
         grid: bool = False,
-        vmin: sc.Variable | float = None,
-        vmax: sc.Variable | float = None,
-        autoscale: Literal['auto', 'grow'] = 'auto',
+        # bbox: BoundingBox | None = None,
+        user_vmin: sc.Variable | float = None,
+        user_vmax: sc.Variable | float = None,
+        # autoscale: Literal['auto', 'grow'] = 'auto',
```

Comment:
While it does seems unrelated, but it was actually necessary to move the autoscaling mechanism from the canvas to the `GraphicalView`, to unify the behaviors between the 2d and 3d canvas, because everything now goes through the same `GraphicalView` (there is no more `Scatter3dView`).

I'll clean it up and then can do a walkthrough to help with the review process.